includes/Core/Admin/Screens.php

```diff
@@ -65,18 +73,21 @@ final class Screens {
 	 *
 	 * @since 1.0.0
 	 *
-	 * @param Context $context Plugin context.
-	 * @param Assets  $assets  Optional. Assets API instance. Default is a new instance.
-	 * @param Modules $modules Optional. Modules instance. Default is a new instance.
+	 * @param Context        $context Plugin context.
+	 * @param Assets         $assets  Optional. Assets API instance. Default is a new instance.
+	 * @param Authentication $authentication  Optional. Authentication instance. Default is a new instance.
+	 * @param Modules        $modules Optional. Modules instance. Default is a new instance.
 	 */
 	public function __construct(
 		Context $context,
 		Assets $assets = null,
+		Authentication $authentication = null,
```

Comment:
As discussed on our call, please update this to be the 4th/next dependency for the constructor rather than changing the 3rd 👍